frontend/src/component/LinkForm.tsx

```diff
@@ -0,0 +1,37 @@
+import {FormEvent, useState} from "react";
+import axios from "axios";
+
+export default function LinkForm() {
+
+    const [file, setFile] = useState<File | undefined>(undefined);
+
+    const handleFileUpload = async (event: FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        if (!file) {
+            alert('Bitte wähle eine Datei aus.');
+            return;
+        }
+        const formData = new FormData();
+        formData.append('file', file);
+
+        axios.post("/api/upload", formData, {
+            headers: { 'Content-Type': 'multipart/form-data' },
+        })
+            .then(response => console.log(response))
+            .catch(error => {console.error(error)});
+    }
```

Comment:
Might be useful to add success and error messages for feedback. 

```suggestion
  try {
        const response = await axios.post("/api/upload", formData, {
            headers: { 'Content-Type': 'multipart/form-data' },
        });
        setMessage("Upload erfolgreich: " + response.data);
    } catch (error) {
        setMessage("Fehler beim Upload.");
        console.error(error);
    }
```